src/server/helpers/props.js

```diff
@@ -27,5 +27,5 @@ export function escapeProps(props) {
 }
 
 export function generateProps(req, res, props) {
-	return Object.assign({}, req.app.locals, res.locals, props);
+	return Object.assign({}, res.locals, props);
```

Comment:
I took this opportunity to pull out `req.app.locals`. There's a ton of information in there that's not relevant to the user and I don't want to risk anything sensitive being added.

Reply:
have you tested this out on all routes? I believe there were some request variables that were being used.

Reply:
I haven't tested it on all routes, but as far as I can tell, the only things which were being pulled from there were site revision and git repo URL. I've moved both of these into `res.locals`.